apps/system/js/utility_tray.js

```diff
@@ -49,7 +49,16 @@ window.UtilityTray = {
   blur: function() {},
 
   init: function ut_init() {
-    var touchEvents = ['touchstart', 'touchmove', 'touchend'];
+    var touchEvents = [
+      'touchstart',
+      'touchmove',
+      'touchend',
+      'click',
```

Comment:
Why do we need click and mouse* events here? I believe that if we call preventDefault() on the event, we should also prevent the mouse events from firing, correct?